unlock-app/src/components/interface/Error.js

```diff
@@ -6,52 +6,45 @@ import { resetError } from '../../actions/error'
 import Buttons from './buttons/layout'
 import ErrorMessage from '../helpers/ErrorMessage'
 
-export const Error = ({ error, close }) => {
```

Comment:
Since we're using `Error`, probably worth renaming the file for consistency?

Reply:
sure 👍